js/index.js

```diff
@@ -1 +1,77 @@
 // Your code goes here
+
+funBusHead = document.querySelector('.logo-heading');
+navBar = document.querySelector('.nav');
+navItems = document.querySelectorAll('.nav-link');
+headerH2 = document.querySelector('.intro h2');
+allParagraphs = document.querySelectorAll('p');
+allExpeditionsTitle = document.querySelectorAll('.destination h4');
+footer = document.querySelector('.footer');
+
+funBusHeading.addEventListener('mouseover', function () {
```

Comment:
Throws an error because funBusHeading is not defined. Perhaps you forgot to do the querySelector for this event listener?